### PR TITLE
ACM-2063: Add host inventory quickstart

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>

https://issues.redhat.com/projects/ACM/issues/ACM-2063

### CODE REVIEW

**Comment by chenz4027** [comment]
@matthewcarleton FYI This is the quick start for host inventory

File: quickstarts/host-inventory-quickstart.yaml
Comment (chenz4027) [inline quickstarts/host-inventory-quickstart.yaml:14]: @jhadvig @spadgett Is there a way to link "OpenShift Data Foundation" to the quick start "quickstart?quickstart=odf-install-tour"?

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

File: quickstarts/host-inventory-quickstart.yaml
Comment (swopebe) [inline quickstarts/host-inventory-quickstart.yaml:68]: Here we are using punctuation at times, and then not at times. All complete sentences should have a period at the end, like the questions and other statements do. (Example, like 68)

File: quickstarts/host-inventory-quickstart.yaml
Comment (swopebe) [inline quickstarts/host-inventory-quickstart.yaml:15]: I mentioned this earlier but didn't look it up....let me check on eg usage.

Comment (swopebe) [reply quickstarts/host-inventory-quickstart.yaml:15]: This is style manual guidance for e.g.:


Do not use e, e-   adjective
Do not create terms using the prefix "e" or "e-".

Comment (swopebe) [reply quickstarts/host-inventory-quickstart.yaml:15]: CSS | Stands for Cascading Style Sheets. Write CSS in all caps.
-- | --
e.g., i.e., and etc. | Use sparingly. These terms aren’t easily understood by everyone, especially users whose native language is not rooted in Latin. Write out their meaning instead:e.g. – for examplei.e. – in other wordsetc. – and more/so on

IBM style is the parent, says no. PatternFly also warns against it...


Let's instead write:

**Example:**

File: quickstarts/host-inventory-quickstart.yaml
Comment (swopebe) [reply quickstarts/host-inventory-quickstart.yaml:68]: the only other thing I will mention is we don't use quotation marks in doc for accessibility reasons, I am not sure how the UI highlighting works, though.
[10:45](https://coreos.slack.com/archives/D013DPB9RTK/p1669736755197489)
and if the OCP team is using it, then I say go forth but we may want to bring that up.

Comment (swopebe) [reply quickstarts/host-inventory-quickstart.yaml:68]: https://www.patternfly.org/v4/ux-writing/punctuation#quotation-marks

File: quickstarts/host-inventory-quickstart.yaml
Comment (chenz4027) [reply quickstarts/host-inventory-quickstart.yaml]: Thanks @swopebe - changes are made

**Comment by swopebe** [comment]
`doc-approved` though I don't have rights to add label, it seems.

**Comment by chenz4027** [comment]
Copying from other PRs:

/hold

QE Approver:
 /assign @yanpzhan
Docs Approver:
 /assign @opayne1
PX Approver:
/assign @RickJWagner

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**
